src/tests/resistance/random_bytes.rs

```diff
@@ -0,0 +1,86 @@
+use tempfile::TempDir;
+use tokio::time::{sleep, Duration};
+
+use crate::{
+    protocol::codecs::payload::Payload,
+    setup::node::Node,
+    tools::{
+        constants::{
+            ERR_NODE_ADDR, ERR_NODE_BUILD, ERR_NODE_CONNECT, ERR_NODE_STOP, ERR_SYNTH_BUILD,
+            ERR_SYNTH_UNICAST, ERR_TEMPDIR_NEW,
+        },
+        synthetic_node::SyntheticNodeBuilder,
+        util::gen_rand_bytes,
+    },
+};
+
+/// Send some randomly generated to the node before the handshake and check the connection status.
```

Comment:
Perhaps it's missing `bytes` (or `data`) in this sentence, but it's clear enough.

Reply:
Done